src/workloads/execution/MultiPlanning.yml

```diff
@@ -107,7 +107,7 @@ AutoRun:
           - standalone
           - standalone-80-feature-flags
           - standalone-all-feature-flags
-          - standalone-classic-query-engine
+          - query-engine-classic
           - standalone-sbe
```

Comment:
Is it intentional to keep `standalone-sbe` here and not change it to `replica-query-engine-sbe`? Also are we purposely keeping this test running on standalone + replicas?

Reply:
Ah, I wasn't sure what to do about this, but I'll change it over for now and we can undo that if someone becomes upset.